packages/fuels-core/src/types/bits_256.rs

```diff
@@ -50,19 +50,80 @@ impl Tokenizable for Bits256 {
     }
 }
 
+#[derive(Debug, PartialEq, Eq, Copy, Clone)]
+// ANCHOR: evm_address
+pub struct EvmAddress {
+    value: Bits256,
+}
+// ANCHOR_END: evm_address
+
+impl EvmAddress {
+    fn clear_12_bytes(bytes: [u8; 32]) -> [u8; 32] {
```

Comment:
Since it's private might save space and just make it a fn instead of an associated fn.

Goes without saying that it doesn't make any difference functionally.

Reply:
I like that it's an associated function since it's a very specific "utility", it would be weird to see it implemented outside this context.